<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PPMainWindow</class>
 <widget class="QMainWindow" name="PPMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Frame"/>
   </widget>
   <widget class="QMenu" name="menuTools_2">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_camera_reset"/>
    <addaction name="actionRender_Frame"/>
    <addaction name="actionRender_All"/>
    <addaction name="actionGenerate_Script"/>
    <addaction name="actionShow_Wind"/>
    <addaction name="actionLoad_Selected_Frame"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Frame"/>
   <addaction name="actionShow_Wind"/>
  </widget>
  <action name="action_camera_reset">
   <property name="text">
    <string>Reset Camera</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionOpen_Frame">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/gui/icons/open-file-folder-svgrepo-com.svg</normaloff>:/gui/icons/open-file-folder-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>Open Frame...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRender_Frame">
   <property name="text">
    <string>Render Frame</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionRender_All">
   <property name="text">
    <string>Render All</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionGenerate_Script">
   <property name="text">
    <string>Generate Script</string>
   </property>
  </action>
  <action name="actionShow_Wind">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="weather-fog"/>
   </property>
   <property name="text">
    <string>Show Wind</string>
   </property>
  </action>
  <action name="actionLoad_Selected_Frame">
   <property name="text">
    <string>Load Selected Frame</string>
   </property>
   <property name="toolTip">
    <string>Load Selected Frame</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
